build_conf.py

```diff
@@ -221,13 +221,13 @@ def set_work_config(self):
             config.read(self.__args.config_file)
             uri = config.get(CFG_SECTION_PATH, CFG_OPTION_WORKSPACE_DIR, 1)
             if uri:
-                self.__workspace_base_dir = uri
+                self.__workspace_base_dir = os.path.expandvars(uri)
```

Comment:
probably you also want to use `expanduser` and `normpath`. 
Also, it would be cool to move suggested callchain to a separate function